website/docs/patterns/filter-patterns/partials/guidelines/guidelines.md

```diff
@@ -163,6 +163,35 @@ If no filters have been applied, use a [Tooltip](/components/tooltip) coupled wi
 
 ![Empty state in applied filters](/assets/patterns/filter-patterns/applied-filters-empty-state.png =450x*)
 
+### Avoiding empty state
+
+Because filtering is often reductive in nature and driven by a user action, it might not always be possible to avoid an empty state within a data set. This can be helped by leveraging specific component variants and properties, and by taking steps to prevent mistakes or errors that would result in an empty state prior to them occurring.
+
+#### ListItem count
+
+Use the `count` property within the Dropdown ListItem to more clearly explicitly call attention to filters that will return zero results.
+
+In this example, `status = pending` does not correspond with any results.
+
+![Count property in ListItem](/assets/patterns/filter-patterns/avoid-empty-state-list-item-count.png)
+
+#### Approximate matching
+
+To help resolve basic spelling mistakes, typos, and broaden the relevancy of a search term being filtered upon, consider using _approximate_ matching or "fuzzy search" when filtering by a string value.
+
+This can help reduce the burden on the user to accurately enter long and complex strings that may consist of seemingly random sequences of characters which are common characteristics of string values within a data set.
+
+!!! Info
+
+More details around the specifics of fuzzy search are beyond the scope of these guidelines, but are included to highlight a creative and user-centric search experience when filtering.
+!!!
+
+### Reverting filters
+
+If the filters applied to a data set _do_ result in an empty state or no results being returned, provide the user with an escape route to revert or all filters at once or individually.
+
+This can be done by highlighting a "Clear all" function in the [Application State](/components/application-state) to bulk clear applied filters, or individually by using a dismissible [Tag](/components/tag).
```

Comment:
[nit] This read a bit odd to me, so I tried reframing a little.

```suggestion
This can be done by adding a "Clear all" action to the [Application State](/components/application-state) that clears all applied filters at once or by including dismissible [Tags](/components/tag) for each applied filter that can be cleared by the user individually.
```